css/mistica-common.css

```diff
@@ -76,20 +178,30 @@
     font-weight: var(--mistica-font-weight-link);
 }
 .mistica-text-title1 {
+    color: var(--vcolor-textSecondary);
+    text-transform: uppercase;
     font-size: var(--mistica-font-size-title1);
```

Comment:
this font-size is wrong

in mobile this size is 12px in desktop 14px. And now is showign 16px in both breakpoints
https://tinyurl.com/24prctjb

Reply:
fixed